tests/contest/runtimetest/src/tests.rs

```diff
@@ -376,3 +380,65 @@ pub fn validate_scheduler_policy(spec: &Spec) {
         eprintln!("error due to sched_nice want {want_sn}, got {sn}")
     }
 }
+
+pub fn test_io_priority_class(spec: &Spec, io_priority_class: IOPriorityClass) {
+    let io_priority_spec = spec
+        .process()
+        .as_ref()
+        .unwrap()
+        .io_priority()
+        .as_ref()
+        .unwrap();
+    if io_priority_spec.class() != io_priority_class {
+        let io_class = io_priority_spec.class();
+        return eprintln!("error io_priority class want {IoprioClassRt:?}, got {io_class:?}");
```

Comment:
I think this needs to be changed for something like this, got left over from merging functions...
```suggestion
        return eprintln!("error io_priority class want {io_priority_spec.class():?}, got {io_class:?}");
```